packages/flet/lib/src/controls/screenshot.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:screenshot/screenshot.dart';
+
+import '../extensions/control.dart';
+import '../models/control.dart';
+import 'base_controls.dart';
+
+class ScreenshotControl extends StatefulWidget {
+  final Control control;
+
+  ScreenshotControl({Key? key, required this.control})
+      : super(key: key ?? ValueKey("control_${control.id}"));
+
+  @override
+  State<ScreenshotControl> createState() => _InteractiveViewerControlState();
+}
+
+class _InteractiveViewerControlState extends State<ScreenshotControl> {
+  final ScreenshotController _screenshotController = ScreenshotController();
+
+  @override
+  void initState() {
+    super.initState();
+    widget.control.addInvokeMethodListener(_invokeMethod);
+  }
+
+  Future<dynamic> _invokeMethod(String name, dynamic args) async {
+    debugPrint("Screenshot.$name($args)");
+    switch (name) {
+      case "capture":
+        return await _screenshotController.capture(
+            pixelRatio: args["pixel_ratio"],
+            delay: args["delay"] ?? const Duration(milliseconds: 20));
```

Comment:
if delay is provided on py-end as a `Duration` object, this will fail right?
I think this line should be like: `parseDuration(args["delay"], const Duration(..))!`